meson.build

```diff
@@ -0,0 +1,1629 @@
+endif
+
+if coreaudio['use'] or audiounit['use']
+    sources += ['audio/out/ao_coreaudio_chmap.c',
+                'audio/out/ao_coreaudio_utils.c']
+endif
+
+jack = dependency('jack', required: get_option('jack'))
+if jack.found() and get_option('gpl').enabled()
+    dependencies += jack
+    sources += 'audio/out/ao_jack.c'
+endif
+
+openal = dependency('openal', version: '>= 1.13', required: get_option('openal'))
+if openal.found()
+    dependencies += openal
+    sources += 'audio/out/ao_openal.c'
+endif
+
+opensles = {
+    'name': 'opensles',
+    'deps': cc.find_library('OpenSLES', required: get_option('opensles')),
+    'statement': run_command('cat', 'waftools/fragments/opensles.c').stdout(),
+    'use': false,
+}
+if opensles['deps'].found()
+    opensles += {'use': cc.compiles(opensles['statement'], dependencies: opensles, name: 'opensles check')}
+endif
+if opensles['use']
+    dependencies += opensles['deps']
+    sources += 'audio/out/ao_opensles.c'
+endif
+
+oss = {
+    'name': 'oss',
+    'statement': run_command('cat', 'waftools/fragments/oss.c').stdout(),
+    'use': false,
+}
+if not get_option('oss-audio').disabled()
+    oss += {'use': cc.compiles(oss['statement'], name: 'oss check')}
+endif
+if oss['use']
+    sources += 'audio/out/ao_oss.c'
+endif
+
+pulse = dependency('libpulse', version: '>= 1.0', required: get_option('pulse'))
+if pulse.found()
+    dependencies += pulse
+    sources += 'audio/out/ao_pulse.c'
+endif
+
+wasapi = {
+    'name': 'wasapi',
+    'statement': run_command('cat', 'waftools/fragments/wasapi.c').stdout(),
+    'use': false,
+}
+if not get_option('wasapi').disabled() and win32
+    wasapi += {'use': cc.compiles(wasapi['statement'], name: 'wasapi check')}
+endif
+if wasapi['use']
+    sources += ['audio/out/ao_wasapi.c',
+                'audio/out/ao_wasapi_changenotify.c',
+                'audio/out/ao_wasapi_utils.c']
+endif
+
+
+# video output dependencies
+caca = dependency('caca', version: '>= 0.99.beta18', required: get_option('caca'))
+if caca.found()
+    dependencies += caca
+    sources += 'video/out/vo_caca.c'
+endif
+
+drm = {
+    'name': 'drm',
+    'deps': dependency('libdrm', version: '>= 2.4.74', required: get_option('drm')),
+    'header': vt_h['use'] or consio_h['use'],
+}
+drm += {'use': drm['deps'].found() and drm['header']}
+if drm['use']
+    dependencies += drm['deps']
+    sources += ['video/out/drm_atomic.c',
+                'video/out/drm_common.c',
+                'video/out/drm_prime.c',
+                'video/out/opengl/hwdec_drmprime_drm.c',
+                'video/out/vo_drm.c']
+endif
+
+gbm = {
+    'name': 'gbm',
+    'deps': dependency('gbm', required: get_option('gbm')),
+    'header': cc.check_header('stdio.h') and cc.check_header('gbm.h'),
+}
+gbm += {'use': gbm['deps'].found() and gbm['header']}
+if gbm['use']
+    dependencies += gbm['deps']
+endif
+
+jpeg = cc.find_library('jpeg', required: get_option('jpeg'))
```

Comment:
libjpeg-turbo (as well as the libjpeg from the IJG, even though no one actually uses that) provides libjpeg.pc via pkg-config, so you can use dependency() here.